### DISCUSSION TITLE
Updated Chia Software - Lost my coins

### DISCUSSION THREAD

**Discussion by LearnedVector** [Chia-Network/chia-blockchain#5024] [Support]
I lost coins that i've farmed a couple months ago when i updated the chia software. My Node and Wallet is fully synced. Anyway I can debug and get these coins back?

**Comment by cccat6** [comment]
Make sure you are using the same private key and your wallet is synced. I mean "Wallet" not "Full Node".
![image](https://user-images.githubusercontent.com/22387156/118349441-3cc7c380-b583-11eb-858d-53afc31be5ac.png)

**Comment by lvcivs** [comment]
A couple months ago? Are you sure it was after mainnet launch?

**Reply by cccat6** [reply]
Nice catch hahaha. I guess he might talking about TXCH.

**Comment by BrandtH22** [comment]
If you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
